docs/releases/2.2.1.txt

```diff
@@ -25,3 +25,7 @@ Bugfixes
 
 * Fixed a regression in Django 2.2 that caused a single instance fast-delete
   to not set the primary key to ``None`` (:ticket:`30330`).
+
+* Fixed the recreation of check constraints (and possibly other operations)
+  when the check expression contained a ``range`` object instead of a list or
```

Comment:
`when the check expression contained a...` -> `when ``condition`` contains a ...`.

I think we can chop `instead of a list or tuple`.

Reply:
I think you could use `:class:`~python:range` as well.